catalog/ui/search-ui/simple/src/main/resources/SearchPage.jsp

```diff
@@ -475,5 +475,29 @@
 
  	<script type="text/javascript" src="js/Search-min.js?bust=${timestamp}"></script>
 
+    <!-- Usage Modal -->
+    <!-- Only show if the modal is enabled -->
+    <% if(props.getSystemUsageEnabled() == true) { %>
```

Comment:
When `systemUsageEnabled` is selected to `true` in the configuration, then run this script and display this modal, otherwise, keep it hidden. 

This is at the bottom of the page because we need jQuery here.